code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -3724,17 +3724,17 @@
 	bitesize = 2
 
 /obj/item/weapon/reagent_containers/food/snacks/gigapuddi/happy
-	desc = "A large crème caramel made with extra love"
+	desc = "A large cr�me caramel made with extra love"
```

Comment:
The accented e changed to a A:: when I added in the crème de la pire, so tried to change them back, seems they've gone weird